### PR TITLE
[terraform-vpc-peerings] fixes for VPC ID

### PR DESCRIPTION
**Pull request by maorfr**
this PR adds some fixes as per #808.

@rporres can you review?

### CODE REVIEW

File: utils/aws_api.py [hunk 1]
```diff
@@ -570,6 +570,25 @@ def init_ecr_auth_tokens(self, accounts):
...
+                                        to the cluster's AWS account
+                        - assume_region - region in which to operate
+                        - assume_cidr - CIDR block of the cluster to
+                                        use to find the matching VPC
+        """
+        required_keys = \
+            ['name', 'assume_role', 'assume_region', 'assume_cidr']
+        ok = all(elem in account.keys() for elem in required_keys)
+        if not ok:
+            account_name = account.get('name')
```
Comment (rporres) [inline utils/aws_api.py:588]: nit: being strict, you may not have `name` here 😅

Comment (maorfr) [reply utils/aws_api.py:588]: i know, but it's the best we can do...

Comment (rporres) [reply utils/aws_api.py:588]: What do you mean by "the best we can do"? If you cannot be sure that you won't have `name` in the input dict, you cannot place it there (unless add more conditions)

Comment (maorfr) [reply utils/aws_api.py:588]: well, `name` is a mandatory field in the schema. it will be here for sure.

**Pull request closed without merging**